### ISSUE TITLE
new option --loop

### ISSUE THREAD

**Issue by jweu** [huceke/omxplayer#195]
art-video-projection at an art space: When we stop and start omxplayer, the raspberry may change the video timing and the beamer is flickering around. Is it a good idea to have a --loop option so that one or more videos loops and omxplayer never stops? At the end, the video should loop from startup until power down.

**Comment by fergardi** [comment]
I totally agree with the OP. This option should be mandatory, and I'll be glad to get it eventually.

Keep up the good job!

**Comment by stewiem2000** [comment]
Assuming that popcornmix's version is the new "official" repo, I've had a go (from scratch) at implementing seamless looping of the current-file in a branch here: https://github.com/stewiem2000/omxplayer/tree/seamless-looping

Hope it helps you both.

**Comment by Carlo69** [comment]
I tried the seamless-looping omxplayer but when I run ./omplayer I receive a "Illegal Instruction".
Also compiling directly on the PI was not really smooth as Libs where not pointed to correct directories.

**Comment by DrakaSAN** [comment]
I m using a (lot older) other branch which support --loop option, the bin are downloadable here: http://pasky.or.cz/dev/omxplayer/omxplayer-loop-multifile-bin.tar.gz

--loop should really be in the main branch, even a lesser quality one.

**Comment by matthijskooijman** [comment]
I think `--loop` is now supported in the main branch, at least in the version shipped in Raspbian Buster, so I think this issue can be closed.

Note that `--loop` works for looping single files. For looping a playlist of files, see #16.
